addons/kube-proxy/daemonset.yaml

```diff
@@ -35,10 +35,10 @@ spec:
       priorityClassName: system-node-critical
       containers:
       - name: kube-proxy
-        image: '{{ Registry "gcr.io" }}/google_containers/hyperkube-amd64:v{{ .Cluster.Version }}'
```

Comment:
It seems like we use `google_containers` in a lot of different places. Can you create a follow-up issue to replace `gcr.io/google_containers` with `k8s.gcr.io/`?

Reply:
https://github.com/kubermatic/kubermatic/issues/5773